src/components/global/header/header.tsx

```diff
@@ -57,71 +73,63 @@ export function Header({ showMarquee = true }: { showMarquee?: boolean }) {
-                                            >
-                                                APP
-                                            </Menu.Button>
-                                            <Transition
-                                                as={Fragment}
-                                                enter="transition ease-out duration-0"
-                                                enterFrom="transform opacity-0 scale-95"
-                                                enterTo="transform opacity-100 scale-100"
-                                                leave="transition ease-in duration-0"
-                                                leaveFrom="transform opacity-100 scale-100"
-                                                leaveTo="transform opacity-0 scale-95"
-                                            >
-                                                <Menu.Items className="absolute left-0 z-10 w-48 origin-top-right bg-black p-0 text-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
-                                                    {/* <img
-                                                        src={triangle_svg.src}
-                                                        className="absolute -top-4 left-8 h-4 w-4"
-                                                    /> */}
+                                        <Popover className={'h-full'}>
+                                            {({ open }) => (
+                                                <div
+                                                    onMouseEnter={() => onHover(open, 'onMouseEnter')}
+                                                    onMouseLeave={() => onHover(open, 'onMouseLeave')}
+                                                    className="relative h-full items-center justify-center "
+                                                >
+                                                    <Popover.Button
+                                                        as="a"
+                                                        className="flex h-full cursor-pointer items-center px-1 text-base font-bold uppercase text-white no-underline hover:bg-white hover:text-black md:px-8"
+                                                        ref={buttonRef}
+                                                    >
+                                                        APP
+                                                    </Popover.Button>
 
-                                                    <Menu.Item>
-                                                        {({ active }) => (
+                                                    <Transition
+                                                        show={open}
+                                                        as={Fragment}
+                                                        enter="transition ease-out duration-0"
```

Comment:
duration-0 makes this immediate. right? that's intended, no?

Reply:
yep, exactly. If the duration is higher, it takes x time to open/close the dropdown